Symfony/CS/Fixer/Contrib/PhpUnitNoDedicateAssertFixer.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author SpacePossum
```

Comment:
https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#72-author
`This tag MAY also contain an e-mail address.`

We had this discussion before, I don't see what is changed in the mean time that justifies having it again?